wealth-watch-script.sql

```diff
@@ -0,0 +1,35 @@
+create table user
+(
+id int unsigned not null unique auto_increment,
```

Comment:
Let's remove the `unique`, we don't need it when it's a primary key.

Reply:
done.